### DISCUSSION TITLE
Change table header in Relation manager

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6163] [Help] [answered]
This may be very easy, but I cannot find how to change.

How to change table heading (title) in relation manager

**Comment by danharrin** [comment]
maybe set the `$title` property?

**Reply by valpuia** [reply]
I can change table header like below

```
public static function getTitle(): string
{
    return __('message.translation');
}
```

But the modal title in create new pop up, title cannot change.. can I translate both of them in one line or another?

![Screenshot from 2023-04-11 09-31-22](https://user-images.githubusercontent.com/28250303/231040347-2f6ea1fa-122a-4bb8-a7bb-35fe6fd84165.png)

**Reply by danharrin** [reply, marked as answer]
You can change the `->modalHeading()` on any action

**Comment by alinme** [comment]
Yes but it needs to be translated

**Comment by alinme** [comment]
You cant do protected static ?string $title = __('New Title');

**Reply by danharrin** [reply]
You override getTitle() then, PHP doesnt let you execute functions in property definitions

**Comment by alinme** [comment]
So how do you do that ?

**Comment by alinme** [comment]
Because you get an error if you do that in the relation manager
